package.json

```
@@ -10,7 +10,7 @@
 		"build:services": "turbo run build --filter=rocketchat-services...",
 		"build:ci": "turbo run build:ci",
 		"testunit": "turbo run testunit",
-		"dev": "turbo run dev --parallel --filter=@rocket.chat/meteor",
+		"dev": "turbo run dev --parallel --filter=@rocket.chat/meteor...",
```

they can't be in parallel since they depend on each other.
```suggestion
		"dev": "turbo run dev --filter=@rocket.chat/meteor...",
```